profiles/views.py

```diff
@@ -220,3 +221,41 @@ def form_valid(self, form):
 
 def about(request, template_name='about.html'):
     return render(request, template_name,)
+
+
+class BulkUser(LoginRequiredMixin, View):
+    '''
+    Custom class to create bulk student users.
+    '''
+
+    def get(self, request):
+        username = request.user
+
+        if username.is_superuser:
+            return render(request, "bulkuserform.html")
+        else:
+            return render(request, "404.html")
+
+    def post(self, request):
+        username = request.user
+
+        if username.is_superuser:
+            format = request.POST["format"]
+            user_start = int(request.POST["start"])
+            user_end = int(request.POST["end"])
+            branch = request.POST["branch"]
+            year = request.POST["year"]
+            course = request.POST["course"]
+            group = Group.objects.get(name="student")
+
+            if user_start < user_end:
+                user_no = ["%.3d" % users for users in range(user_start, user_end+1)]
+                for student in user_no:
+                    new_user = User.objects.create_user(username=format+str(student), password=str(format)+str(student))
+                    group.user_set.add(new_user)
+                    StudentDetail.objects.create(user=new_user, branch=branch.upper(), year=year, course=course)
+                return render(request, "bulkuser.html")
+            else:
+                return render(request, "bulkuserform.html", {"error": 1})
```

Comment:
Do we really need two separate templates `bulkuser.html` and `bulkuserform.html` ?? Can you explain the significance of these two ?